frontend/src/core/runtime/runtime.ts

```diff
@@ -135,13 +135,7 @@ export class RuntimeManager {
   /**
    * The URL of the copilot server.
    */
-  getLSPURL(lsp: "pylsp" | "copilot"): URL {
-    if (lsp === "copilot") {
-      // For copilot, don't include any query parameters
-      const url = this.formatWsURL(`/lsp/${lsp}`);
-      url.search = "";
-      return url;
-    }
+  getLSPURL(lsp: "pylsp" | "copilot" | "ty"): URL {
```

Comment:
`formatWsURL` might. and its safer and clearer anyways. this was added because of a bug